packages/server/src/db/schema.ts

```diff
@@ -9,3 +10,17 @@ export const users = table("users", {
 	passwordHash: t.text("password_hash").notNull(),
 	createdAt: t.integer("created_at", { mode: "timestamp" }).notNull(),
 });
+
+export const question = table("question", {
+	postId: t.integer("post_id").primaryKey({ autoIncrement: true }),
```

Comment:
```suggestion
	postId: t.text("post_id").primaryKey(),
```
idはIdPだとuuid使ってるからそっちに合わせようと思う。調べた感じuuid型はないっぽいからtextでいこう。
https://orm.drizzle.team/docs/column-types/sqlite